src/init.cpp

```diff
@@ -188,7 +188,7 @@ void Shutdown()
     if (!lockShutdown)
         return;
 
-    /// Note: Shutdown() must be able to handle cases in which AppInit2() failed part of the way,
+    /// Note: Shutdown() must be able to handle cases in which AppInitMain() failed part of the way,
```

Comment:
Done (also in bitcoin.cpp).

For my information: how should I address feedback: with new commits for easier review, or with amending? I amended this time.